pkgs/sketch_pad/lib/model.dart

```diff
@@ -367,7 +356,7 @@ enum Channel {
 
   const Channel(this.displayName, this.url);
 
-  static const defaultChannel = Channel.stable;
+  static const defaultChannel = Channel.localhost;
```

Comment:
This change looks like it was included by accident

Reply:
Yup - removed.